### PR TITLE
Bump to Xcode 13 beta 1.

### PR DESCRIPTION
**Pull request by rolfbjarne**

### CODE REVIEW

File: Make.versions [hunk 1]
```diff
@@ -66,11 +66,11 @@ MAC_PACKAGE_VERSION=7.15.0.$(MAC_COMMIT_DISTANCE)
...
-IOS_NUGET_VERSION=14.5.100
-TVOS_NUGET_VERSION=14.5.100
-WATCHOS_NUGET_VERSION=7.4.100
-MACOS_NUGET_VERSION=11.3.100
-MACCATALYST_NUGET_VERSION=14.5.100
+IOS_NUGET_VERSION=15.0.100
+TVOS_NUGET_VERSION=15.0.100
+WATCHOS_NUGET_VERSION=8.0.100
+MACOS_NUGET_VERSION=12.0.100
+MACCATALYST_NUGET_VERSION=15.0.100
```
Comment (spouliot) [inline Make.versions:73]: For legacy
```
IOS_PACKAGE_VERSION=14.21.0.$(IOS_COMMIT_DISTANCE)
MAC_PACKAGE_VERSION=7.15.0.$(MAC_COMMIT_DISTANCE)
```
also needs to be updated

**Review by spouliot — CHANGES_REQUESTED** [review]

File: Make.versions (hunk 1 above)
Comment (spouliot) [reply Make.versions:73]: Might as well include the maccore bump as the same time

File: Make.config [hunk 2]
```diff
@@ -126,9 +126,9 @@ MACCATALYST_NUGET_VERSION_NO_METADATA=$(MACCATALYST_NUGET_VERSION)-$(NUGET_PRERE
 MACCATALYST_NUGET_VERSION_FULL=$(MACCATALYST_NUGET_VERSION_NO_METADATA)+$(NUGET_BUILD_METADATA)
 
 # Xcode version should have both a major and a minor version (even if the minor version is 0)
-XCODE_VERSION=12.5
-XCODE_URL=https://dl.internalx.com/internal-files/xcodes/Xcode_12.5.xip
-XCODE_DEVELOPER_ROOT=/Applications/Xcode_12.5.0.app/Contents/Developer
+XCODE_VERSION=13.0
+XCODE_URL=https://dl.internalx.com/internal-files/xcodes/Xcode_13.0-beta.xip
```
Comment (spouliot) [inline Make.config:130]: Internal email provided this URL `https://bosstoragemirror.azureedge.net/internal-files/xcodes/Xcode_13_beta.xip`
and it seems to work fine

Comment (spouliot) [reply Make.config:130]: well `dl.internalx.com` should work too - but the filename is different, you have an extra `.0`

File: Make.versions (hunk 1 above)
Comment (rolfbjarne) [reply Make.versions:73]: Fixed (both)

File: Make.config (hunk 2 above)
Comment (rolfbjarne) [reply Make.config]: Fixed too.

**Review by spouliot — APPROVED** [review]

**Review by mandel-macaque — APPROVED** [review]

**Comment by mandel-macaque** [comment]
Those links are fixed with https://github.com/xamarin/xamarin-macios/pull/11865

**Comment by spouliot** [comment]
```
Failed to download https://devimages-cdn.apple.com/downloads/xcode/simulators/index-13.0.0-42E1F17B-27B3-4DE8-92A8-DC76BA4F5921.dvtdownloadableindex: Not found
com.apple.pkg.iPhoneSimulatorSDK11_4
com.apple.pkg.iPhoneSimulatorSDK12_0
com.apple.pkg.AppleTVSimulatorSDK11_4
com.apple.pkg.WatchSimulatorSDK5_0
    ** failure ** Failed to install extra simulators.
```
c.c. @mandel-macaque

**Comment by mandel-macaque** [comment]
Great... github decided to close because I mentioned the Pr in my own PR :/

**Review by stephen-hawley — APPROVED** [review]

**Comment by spouliot** [comment]
```
23:01:50.3301520 clang -m32 -dynamiclib -o /Users/builder/azdo/_work/1/s/xamarin-macios/tests/xharness/tmp-test-dir/mmptest/bin/Debug/tmp-test-dir/Xamarin.MMP.Tests.MMPTests.RunMMPTest51/foo_32.dylib /Users/builder/azdo/_work/1/s/xamarin-macios/tests/xharness/tmp-test-dir/mmptest/bin/Debug/tmp-test-dir/Xamarin.MMP.Tests.MMPTests.RunMMPTest51/foo_32.o
23:01:50.3301630 ld: warning: The i386 architecture is deprecated for macOS (remove from the Xcode build setting: ARCHS)
23:01:50.3301730 ld: warning: ignoring file /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/lib/libSystem.tbd, missing required architecture i386 in file /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/lib/libSystem.tbd (3 slices)
23:01:50.3301990 ld: dynamic main executables must link with libSystem.dylib for architecture i386
23:01:50.3302160 clang: error: linker command failed with exit code 1 (use -v to see invocation)
```
looks like 32bits is really gone

**Comment by rolfbjarne** [comment]
> ```
> 23:01:50.3301520 clang -m32 -dynamiclib -o /Users/builder/azdo/_work/1/s/xamarin-macios/tests/xharness/tmp-test-dir/mmptest/bin/Debug/tmp-test-dir/Xamarin.MMP.Tests.MMPTests.RunMMPTest51/foo_32.dylib /Users/builder/azdo/_work/1/s/xamarin-macios/tests/xharness/tmp-test-dir/mmptest/bin/Debug/tmp-test-dir/Xamarin.MMP.Tests.MMPTests.RunMMPTest51/foo_32.o
> 23:01:50.3301630 ld: warning: The i386 architecture is deprecated for macOS (remove from the Xcode build setting: ARCHS)
> 23:01:50.3301730 ld: warning: ignoring file /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/lib/libSystem.tbd, missing required architecture i386 in file /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/lib/libSystem.tbd (3 slices)
> 23:01:50.3301990 ld: dynamic main executables must link with libSystem.dylib for architecture i386
> 23:01:50.3302160 clang: error: linker command failed with exit code 1 (use -v to see invocation)
> ```
> 
> looks like 32bits is really gone

It doesn't look like that's using the Xcode we want it to use, but it's instead building using the system sdk.

**Comment by spouliot** [comment]
> introspection/watchOS 32-bits - simulator/Debug (watchOS 5.0): Failed

unrelated crash (it happened before Xcode 13) -> https://github.com/xamarin/maccore/issues/2454

**Review by dalexsoto — APPROVED** [review]

**Pull request merged**